backend/src/main/scala/cromwell/backend/standard/callcaching/StandardFileHashingActor.scala

```diff
@@ -91,7 +93,7 @@ abstract class StandardFileHashingActor(standardParams: StandardFileHashingActor
     }
 
     ioHashCommandTry match {
-      case Success(ioHashCommand) => sendIoCommandWithContext(ioHashCommand, fileRequest)
+      case Success(ioHashCommand) => sendIoCommandWithContext(ioHashCommand, FileHashContext(fileRequest.hashKey, fileRequest.file.value))
```

Comment:
Should this be broken into its own PR as it sounds like something we'd definitely want?